### PR TITLE
moc_globe_atl implementation

### PR DESCRIPTION
**Pull request by anissa111**
Addresses #9 

Purpose: implements moc_globe_atl function in geocat f2py efforts

Summary of changes:
- moves MOC.loops.f into 
- creates .pyf file for f2py conversion
- creates wrapper function
- includes and alters previously created tests for moc_globe_atl

### CODE REVIEW

**Review by erogluorhan — CHANGES_REQUESTED** [review]
It looks great overall!

Please update from `master` branch to get the latest version of `build.sh` and `meta.yaml` files. Then add your corresponding `moc_globe_atl` line to `build.sh`.

**Comment by anissa111** [comment]
> Please update from `master` branch to get the latest version of `build.sh` and `meta.yaml` files. Then add your corresponding `moc_globe_atl` line to `build.sh`.

Addressed, thank you for the catch!

**Review by erogluorhan — APPROVED** [review]
It looks great and I am approving it. Just out of curiosity: It looks like test cases `test_moc_globe_atl_msg_nan_float32` and `test_moc_globe_atl_msg_99_float32` have been dropped; were they abundant in the existence of `test_moc_globe_atl_msg_nan` and `test_moc_globe_atl_msg_99?

File: src/geocat/temp/__init__.py [hunk 1]
```diff
@@ -2,4 +2,5 @@
 from .errors import *
 from .linint2_wrapper import (linint1, linint2, linint2pts)
 from .rcm2rgrid_wrapper import (rcm2rgrid, rgrid2rcm)
+from .moc_globe_atl_wrapper import moc_globe_atl
```
Comment (pilotchute) [inline src/geocat/temp/__init__.py:5]: I'm going to add parentheses on master once merged

**Review by pilotchute — APPROVED** [review]

**Pull request merged**